google-cloud-clients/google-cloud-contrib/google-cloud-logging-logback/src/main/java/com/google/cloud/logging/logback/LoggingAppender.java

```diff
@@ -228,13 +239,6 @@ public synchronized void stop() {
   }
 
   Logging getLogging() {
-    if (logging == null) {
-      synchronized (this) {
-        if (logging == null) {
-          logging = getLoggingOptions().getService();
-        }
-      }
-    }
```

Comment:
where logging service would be created? Public constructor dont have logging parameter and other constructor with logging parameter is package private.

Reply:
The change is to create the logging service in start.